### ISSUE TITLE
a11y support docs

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-hyperlink#129]
# General Support Request

The question seems to come up again and again that the way Auro designates its hyperlinks is inaccessible. 

## Support request

Create and publish supporting documentation that defines auro-hyperlink's WCAG perspective.
